openpype/modules/deadline/abstract_submit_deadline.py

```diff
@@ -627,7 +628,14 @@ def submit(self, payload):
             self.log.debug(payload)
             raise RuntimeError(response.text)
 
-        result = response.json()
+        try:
+            result = response.json()
+        except json.decoder.JSONDecodeError:
+            msg = "Broken response {}. ".format(response)
+            msg += "Try restarting DL webservice"
```

Comment:
I think abbreviating to Deadline to DL is just adding confusion in the clarity of the message.

```suggestion
            msg += "Try restarting the Deadline Webservice."
```